src/scripting/events.cpp

```diff
@@ -112,6 +112,10 @@ void trigger_national_event(sys::state& state, dcon::national_event_id e, dcon::
 	if(ft == slot_type::province)
 		assert(dcon::fatten(state.world, state.world.province_get_nation_from_province_ownership(trigger::to_prov(from_slot))).is_valid());
 
+
+	auto name = text::produce_simple_string(state, dcon::fatten(state.world, e).get_name());
```

Comment:
it looks like you left some debugging stuff in here

Reply:
Fixed w #13ae911